packages/lesswrong/components/users/CrosspostLoginPage.tsx

```diff
@@ -94,7 +96,7 @@ const CrosspostLoginPage = ({classes}: {
           </>
         )
         : (
-          <LoginForm immediateRedirect />
+          <LoginForm immediateRedirect={!isE2E} />
```

Comment:
Without this it tries to redirect to the off-site auth0 login page, which is obviouslly bad during an E2E test